src/plugins/github/fetchGithubRepo.js

```diff
@@ -126,10 +127,20 @@ function tryGithubFetch(fetch, fetchOptions): Promise<any> {
           }
         }
         if (x.data === undefined) {
-          // See https://github.com/sourcecred/sourcecred/issues/350.
-          return Promise.reject(
-            ({type: "NO_DATA", retry: true, error: x}: GithubResponseError)
-          );
+          // See https://github.com/sourcecred/sourcecred/issues/350
```

Comment:
I think this comment is meant to be bound to the `({type: "NO_DATA", retry: true, ...` line, as #350 essentially justifies why we would try to retry in that particular case. Could you move the comment down a few lines so it's still connected to that line?

Reply:
Good call! Fix incoming.